src/cgame/cg_rocket_events.cpp

```diff
@@ -70,6 +71,24 @@ static void CG_Rocket_EventHide()
 	Rocket_DocumentAction( CG_Argv( 1 ), "hide" );
 }
 
+// Used to enforce only one passthrough menu at a time because if you have two open and close one,
+// the other doesn't grab focus so you can't send inputs. Also the circle menus are
+// transparent so it doesn't look very good to stack them.
+static void HideOtherPassthrough()
+{
+	std::string skip = CG_Argv( 1 );
+
+	for ( const rocketMenu_t &menu : rocketInfo.menu )
+	{
+		if ( menu.passthrough && menu.id && skip != menu.id )
+		{
+			if ( Rml::ElementDocument *document = menuContext->GetDocument( menu.id ) )
+			{
+				document->Hide();
+			}
+		}
+	}
+}
```

Comment:
Can we just implement this in pure lua to avoid the round trip through our event system?

Reply:
Basically I think this could just be a "HideOthers()" function that would close all non

Reply:
I think it's a good idea to use the pushevent thing in case `Hide` or other document functions are not reentrant. Wouldn't want to have the hide function called while the show function is executing or anything like that.

Reply:
Personally, it seems like a HideOthers is way more applicable. In fact, I'm not even sure the passthrough part of this is relevant. You probably want to close *all* other menus except the circle menu that just opened. 

Lastly, if you choose not to make this change, at the very least you shouldn't hard code the id. You should be able to get it by calling `document.id` or the equivalent lua.